R/html_document.R

```diff
@@ -597,20 +674,26 @@ navbar_links_tags <- function(links, depth = 0L) {
           link_text <- navbar_link_text(x, " ", tags$span(class = "caret"))
         }
 
-        submenuLinks <- navbar_links_tags(x$menu, depth = depth + 1L)
-
-        tags$li(class = menu_class,
-                tags$a(
-                  href = "#", class = "dropdown-toggle",
-                  `data-toggle` = "dropdown", role = "button",
-                  `aria-expanded` = "false", link_text),
-                tags$ul(class = "dropdown-menu", role = "menu", submenuLinks)
+        submenuLinks <- navbar_links_tags(x$menu, depth = depth + 1L, version)
+
+        tags$li(
+          class = menu_class,
+          class = if (!bs3_nav) "nav-item",
```

Comment:
Can you double check that dropdown menus work properly when `bootstrap_version: 4`?

Reply:
Pretty sure they work (although, see https://github.com/rstudio/rmarkdown/pull/1706#discussion_r350966939), but it looks like `nav-item` should only be applied to non-dropdowns (https://github.com/rstudio/rmarkdown/pull/1706/commits/66cd4b48f4dcff4024474418f4b76e9f0d5f67f9)